plugin/storage/es/spanstore/reader_test.go

```diff
@@ -89,7 +90,12 @@ func withSpanReader(fn func(r *spanReaderTest)) {
 		client:    client,
 		logger:    logger,
 		logBuffer: logBuffer,
-		reader:    newSpanReader(client, logger, 72*time.Hour, ""),
+		reader: newSpanReader(SpanReaderParams{
```

Comment:
The previous notation is far more readable, IMO.

Reply:
The param struct was requested in https://github.com/jaegertracing/jaeger/pull/1009#discussion_r211639597